index.html

```diff
@@ -395,7 +511,7 @@ <h3 class="contributor__subtitle">
                             <div class="row">
                                 <div class="col-xl-10 offset-xl-1">
                                     <div class="contributor__list-title">
-                                        Over 100 Contributors
+                                        Over 150 Contributors
```

Comment:
cauldron.io reports 228 for the LTS repository alone

Reply:
<img width="438" alt="Screenshot 2023-01-06 alle 11 28 51" src="https://user-images.githubusercontent.com/909743/210988107-515a574d-2f3a-4691-a073-dcf11ad68183.png">

Reply:
but also:

<img width="325" alt="Screenshot 2023-01-06 alle 11 30 26" src="https://user-images.githubusercontent.com/909743/210988689-23f43b33-73bb-4266-9dfc-478ba521893a.png">

Reply:
- 228 seem be all contributors (ignoring name/email) - (i should be listed with at least 3 "accounts")
- 151 added themselve to contributers list
- 177 correct imho